templates/vue/store/modules/foo/create.js

```diff
@@ -0,0 +1,101 @@
+import SubmissionError from '../../../error/SubmissionError';
+import {{{ lc }}}Fetch from '../../../api/{{{ lc }}}Fetch';
```

Comment:
This should use `utils/fetch` directly, the `fooFetch` file is not necessary IMO.

Reply:
I was thinking about that but keep this to allow wrapping... I have no preferences

Reply:
@mysiar wdyt?